packages/core/admin/package.json

```diff
@@ -128,6 +128,7 @@
     "sift": "13.5.4",
     "style-loader": "3.3.1",
     "styled-components": "5.3.3",
+    "typescript": "4.6.2",
```

Comment:
Do we have a hard dependency on typescript?

Reply:
the webpack ts-checker plugin has one apparently